src/devices/Mcp960X/Mcp960X.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.MCP960X
+{
+    /// <summary>
+    /// MCP960X - cold-junction compensated thermocouple to digital converter
+    /// </summary>
+    public class MCP960X : IDisposable
```

Comment:
casing: Mcp960x

Reply:
Rename of class is done. 
Should the name space of be remaned Iot.Device.MCP960X --> Iot.Device.Mcp960x